### PR TITLE
add message detail to the QR-KEX

### PR DESCRIPTION
**Pull request by tomato42**

### CODE REVIEW

File: ietf-gss-kex-qr.xml [hunk 1]
```diff
@@ -102,7 +102,9 @@
     <section title="New Quantum Resistant Key Exchange Methods">
       <t>This document adopts the same naming convention as defined in <xref
         target="RFC4462"/> to define families of methods that cover any
-        GSS-API mechanism used with a specific SHA-2 Hash.
+          GSS-API mechanism used with a specific SHA-2 Hash. It also
+          reuses much of the the scheme defined in Section 2.1 of
```
Comment (sahanaprasad07) [inline ietf-gss-kex-qr.xml:106]: Does 'reuses most of the scheme' sound better? Not sure :)

File: ietf-gss-kex-qr.xml [hunk 2]
```diff
@@ -133,6 +135,198 @@
...
+        <t>This section reuses much of the scheme defined in Section 2.1
+            of <xref target="RFC4462"/> though it does not transport FFDH
+            key shares in the exchanged messages.
+        </t>
+        <t>This section defers to <xref target="RFC7546"/> as the source of
+            information on GSS-API context establishment operations,
+            Section 3 being the most relevant. All security considerations
+            described in <xref target="RFC7546"/> apply here too.
+        </t>
+        <t>The parties generate in the key exchange generate nonces.
```
Comment (sahanaprasad07) [inline ietf-gss-kex-qr.xml]: could there be an extra generate in this sentence

File: ietf-gss-kex-qr.xml [hunk 3]
```diff
@@ -133,6 +135,198 @@
...
+            The generated nonces MUST be at lest 256 bits long and come from a
```
Comment (sahanaprasad07) [inline ietf-gss-kex-qr.xml]: least

File: ietf-gss-kex-qr.xml [hunk 4]
```diff
@@ -133,6 +135,198 @@
...
+            quantum safe CSPRNG. The nonces MUST NOT be reused in other key
+            exchanges.
+        </t>
+        <t>The client initiates negotiation by calling GSS_Init_sec_context()
+            and the server responds to it by calling GSS_Accept_sec_context().
+            For the negotiation client MUST set the mutual_req_flag,
```
Comment (sahanaprasad07) [inline ietf-gss-kex-qr.xml]: For the negotiation,

File: ietf-gss-kex-qr.xml (hunk 1 above)
Comment (tomato42) [reply ietf-gss-kex-qr.xml:106]: yeah, but "most" implies majority, and I don't think we're there :)

File: ietf-gss-kex-qr.xml (hunk 2 above)
Comment (tomato42) [reply ietf-gss-kex-qr.xml]: fixed

File: ietf-gss-kex-qr.xml (hunk 3 above)
Comment (tomato42) [reply ietf-gss-kex-qr.xml]: fixed

**Pull request merged**